src/components/common/query/QueryView.jsx

```diff
@@ -0,0 +1,129 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+
+import { Button, Typography } from '@material-ui/core';
+
+import { Link } from 'react-router-dom';
+import SidebarDrawer from 'components/dataset/data/sidebar/SidebarDrawer';
```

Comment:
the `SidebarDrawer` should be moved out of the `dataset` folder if its used for both datasets and snapshots.

Reply:
Agreed - I think this will be covered in https://broadworkbench.atlassian.net/browse/DR-2483 - I'll add this to A.C.